Content.Server/Voting/Managers/VoteManager.DefaultVotes.cs

```diff
@@ -184,17 +190,24 @@ private void CreateMapVote(IPlayerSession? initiator)
             vote.OnFinished += (_, args) =>
             {
                 GameMapPrototype picked;
+                string title;
                 if (args.Winner == null)
                 {
                     picked = (GameMapPrototype) _random.Pick(args.Winners);
+                    title = maps.FirstOrDefault(x => x.Value == picked).Key;
                     _chatManager.DispatchServerAnnouncement(
-                        Loc.GetString("ui-vote-map-tie", ("picked", maps[picked])));
+                        Loc.GetString("ui-vote-map-tie", ("picked", title)));
                 }
                 else
                 {
                     picked = (GameMapPrototype) args.Winner;
-                    _chatManager.DispatchServerAnnouncement(
-                        Loc.GetString("ui-vote-map-win", ("winner", maps[picked])));
+                    title = maps.FirstOrDefault(x => x.Value == picked).Key;
+                    _chatManager.DispatchServerAnnouncement(Loc.GetString("ui-vote-map-win",
+                        ("winner", title)));
+                }
+                if (title == Loc.GetString("ui-vote-secret-map"))
+                {
+                    _chatManager.DispatchServerAnnouncement(Loc.GetString("ui-vote-secret-win"));
```

Comment:
This seems a lil hacky, I would just leave it as it was, having it generate "Secret won the map vote!" is clear enough and better than using ftl strings as conditions